example/particle_leapfrog/particle_leapfrog.cpp

```diff
@@ -117,174 +118,6 @@ Real EstimateTimestepBlock(MeshBlockData<Real> *rc) {
 
 } // namespace Particles
 
-// Simple log function to ease regression tests and debugging until there's a proper
-// particles output method. Do NOT use in practice.
-TaskStatus WriteParticleLog(BlockList_t &blocks, int ncycle) {
```

Comment:
Oh right this is deprecated :) It performed a good service for a long time

Reply:
Deprecated now that we have real IO, anyway. :)